### PR TITLE
Added basic auth support

### PR DESCRIPTION
**Pull request by mhvelplund**
I needed basic auth support, and I thought I'd share the implementation with you ;)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -13,18 +13,23 @@ Using the task is simple. The only mandatory attribute is url and the default HT
...
-	        contenttype="text/xml"
+	        contenttype="application/json"
 	        connectiontimeout="30"
 	        responseproperty="response"
 	        statuscodeproperty="status"
-	        failonerror="true" />
+	        failonerror="true" 
+	        username="jdoe"
+	        password="53cr34"
+	        content="{ &quot;greeting}&quot; : &quot;Hello world!&quot; }" />
```
Comment (alexmg) [inline Readme.md:25]: Probably doesn't make sense to include `content` on a GET request (looks like there is also an extra curly bracket). Maybe include in a separate POST example?

**Comment by alexmg** [comment]
Don't worry, I'll add a POST example. Thanks again for contributing! :+1:

File: Readme.md (hunk 1 above)
Comment (mhvelplund) [inline Readme.md:25]: Hmm ... true. I'd also need to change the URL to http://httpbin.org/post to keep internal consistency.

Comment (mhvelplund) [inline Readme.md:25]:
```
<?xml version="1.0"?>
<project name="Http">
  <http url="http://httpbin.org/post"
        method="POST"
        contenttype="application/json"
        connectiontimeout="30"
        responseproperty="response"
        statuscodeproperty="status"
        failonerror="true" 
        username="jdoe"
        password="53cr34"
        content="{ &quot;greeting&quot; : &quot;Hello world!&quot; }" />

  <echo message="Response: ${response}" />
  <echo message="Status Code: ${status}" />
</project>
```

**Pull request merged**